js-packages/@prestojs/viewmodel/src/__tests__/ViewModelCache.test.ts

```diff
@@ -151,6 +151,9 @@ test('should validate pk(s)', () => {
     expect(() => Test2.cache.get({ id1: null }, ['name'])).toThrowError(
         'Test2 has a compound key of id1, id2. Missing value(s) for field(s) id1, id2'
     );
+
+    expect(Test2.cache.getAll(['id1', 'id2', 'name'])[0].id1).toBe(record2.id1)
```

Comment:
it should return the exact same record so can just be

```
expect(Test2.cache.getAll(['id1', 'id2', 'name'])[0]).toBe(record2);
```

also can you add case for just passing the 'name':

```
expect(Test2.cache.getAll(['name'])[0]).toBe(record2);
```

(should work - pks should be included automatically)

Reply:
I tried .toBe(record2) initially, but it didn't work. I'll try it again though and post the error.